### ISSUE TITLE
stuck in maintenance mode after update

### ISSUE THREAD

**Issue by jerabaul29** [Nitrokey/nextbox#32]
Hi,

Following the instructions in one of the NextCloud notifications, I accepted to update the NextCloud app running on my NextBox. Since then (this happened a few days ago), I only get the "Maintenance mode" screen. I tried pulling off the power, it does not help. How can I / should I fix this? I guess it is a bit at the boundary of what NextCloud and NextBox provide, as this seems to be a NextCloud issue, but still wandering what the recommended way to handle this on the NextBox case is, given that there are "restrictions" on how to log on the RPi host...

Thanks in advance for the help!

The screenshot that I have been getting for the last few days:

![Screenshot from 2021-12-22 17-53-27](https://user-images.githubusercontent.com/8382834/147128289-a7c25a9d-8c19-4ad3-a9d2-1570db0bc794.png)

**Comment by jerabaul29** [comment]
Ok, after some more checks on the NextBox webpage I found out this: 

Q: My Nextcloud instance is stuck in “Maintenance Mode”, how can I switch it off?
    To force exit the Nextcloud “Maintenance Mode”, you can push the hardware button shortly, once. The NextBox will then switch-off the maintenance mode. Please avoid this, if possible.

here: https://docs.nitrokey.com/nextbox/faq/nextcloud.html?highlight=maintenance . This works, I got access back to the NextBox without loss of data :) .

Just being curious: how / what is implemented / takes place when this button is pressed? Is there a software hook reversing to the last version of the NetCloud container or something like this? :)
